test/js/fields/FileField.unit.spec.jsx

```diff
@@ -43,6 +44,7 @@ describe('Schemaform <FileField>', () => {
     );
 
     expect(tree.find('label').first().text()).to.contain('Upload');
+    expect(tree.find('input').getDOMNode().getAttribute('autocomplete')).to.equal('date');
```

Comment:
other thing to note: we're using the react `autoComplete` as the prop name instead of the the attribute name. I'm sort of on the fence about this but I wanted to avoid going from 

autocomplete (form prop) -> autoComplete (react prop) -> autocomplete (markup)

Reply:
Yeah it's unfortunate that the DOM and HTML use different capitalization. I'd prefer to not be tied to React conventions, so `autocomplete`. When looking the information up about how to set the attribute in [places like this](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete), that's what people will see. So for the form creator it's more intuitive, just a bit more mental shift for React programmers who [already have to deal with a bunch of this](https://reactjs.org/docs/dom-elements.html).

Also, if we change the current `options.autocomplete` to `options.autoComplete` it is a breaking change and we technically shouldn't be doing it in a minor version release.

Would a FileField ever really have an `autocomplete` of `"date"`? I don't think we need to test on *every* widget type as much as ensure that the basic ones like TextWidget are working properly and in particular to be sure we got the camelCase thing fixed.

Reply:
I've updated the prop names to `autocomplete` and removed a few of the unit tests from the widgets @dmethvin-gov